### PR TITLE
Add support for multiple success status codes for toolbar display

### PR DESCRIPTION
**Pull request by Sruthin86**
We encountered a scenario where we need the toolbar to be displayed on additional status codes, so our team added functionality which enables the toolbar display on multiple success status codes.

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -1,3 +1,3 @@
 [submodule "docs/_themes"]
 	path = docs/_themes
-	url = git://github.com/mitsuhiko/flask-sphinx-themes.git
+	url = https://github.com/mitsuhiko/flask-sphinx-themes.git
```
Comment (jeffwidman) [inline .gitmodules:3]: This is out of scope of this PR, feel free to open a different PR with this if you want it, although you'd need to make a convincing argument for it, because the `git` url seems fine to me.

File: flask_debugtoolbar/static/css/toolbar.css [hunk 2]
```diff
@@ -167,6 +167,7 @@
   left:0px;
   background-color:#eee;
   color:#666;
+  overflow:auto;
```
Comment (jeffwidman) [inline flask_debugtoolbar/static/css/toolbar.css:170]: Why? Is this related to the success codes change?

File: flask_debugtoolbar/templates/base.html [hunk 3]
```diff
@@ -1,4 +1,4 @@
-<div id="flDebug" style="display:none;">
+<div role="navigation" id="flDebug" style="display:none;">
```
Comment (jeffwidman) [inline flask_debugtoolbar/templates/base.html:1]: Why? It's been years since I actively worked on browser-facing code, so if this is some new HTML thing that I'm now ignorant of, please enlighten me.

File: flask_debugtoolbar/__init__.py [hunk 4]
```diff
@@ -50,6 +50,7 @@ class DebugToolbarExtension(object):
         os.path.join(os.path.dirname(__file__), 'static'))
 
     _redirect_codes = [301, 302, 303, 304]
+    _success_codes = [200, 201]
```
Comment (jeffwidman) [inline flask_debugtoolbar/__init__.py:53]: Can you provide a little more background on this use case? Why do you want the toolbar visible on a `201`? Is it for the sql pane?

File: flask_debugtoolbar/static/css/toolbar.css (hunk 2 above)
Comment (natecollins) [reply flask_debugtoolbar/static/css/toolbar.css:170]: Nope, it's unrelated - my apologies! Our various commits should have been in different branches, not stacked onto the `master` branch of the cloned repository. This one line was to allow scrolling of panels that overflowed beyond the vertical space in the browser. Go ahead and reject this if you'd like and we can put in a separate PR.

File: flask_debugtoolbar/__init__.py (hunk 4 above)
Comment (natecollins) [reply flask_debugtoolbar/__init__.py:53]: This was mainly for convenient access to log entries generated while debugging, though the SQL and profiler panes may come in handy at some point in the future.

File: .gitmodules (hunk 1 above)
Comment (natecollins) [reply .gitmodules:3]: You are correct - we can certainly put this into a separate PR. For background, it's related to a security change on GitHub that took effect earlier this month ( https://github.blog/2021-09-01-improving-git-protocol-security-github/ ).

**Comment by jeffwidman** [comment]
Why don't you break the relevant bits into separate PR's as it will be easier to triage/discuss the individual topics that way?

File: flask_debugtoolbar/templates/base.html (hunk 3 above)
Comment (natecollins) [reply flask_debugtoolbar/templates/base.html:1]: This can be another separate PR. It's a change to improve the toolbar's accessibility compliance, which is helpful for developers using accessibility tools like a screen reader. HTML5 has native landmark tags (e.g. `<nav>`) to indicate an element's purpose. Using the ARIA `role` attribute is a HTML4 backwards compatible way of doing the same thing.

**Comment by natecollins** [comment]
For sure, Jeff. We'll get these changes split up properly and then re-submit them.

**Comment by Sruthin86** [comment]
Closing this request as our team got in touch with the maintainer and are working towards a solution in a different branch
